src/App/App.test.js

```diff
@@ -1,8 +0,0 @@
-// import { render, screen } from '@testing-library/react';
-// import App from './App';
-
-// test('renders learn react link', () => {
-//   render(<App />);
-//   const linkElement = screen.getByText(/learn react/i);
-//   expect(linkElement).toBeInTheDocument();
-// });
```

Comment:
If we don't want to run Jest tests, we should just delete this file.

Reply:
I tried but it gave me a merge conflict so i had to add it back in